test/test_analysis_ee_Z_bb.py

```diff
@@ -7,17 +7,19 @@
 import heppy.framework.context as context
 
 if context.name == 'fcc':
-
-    #from analysis_ee_Z_bb_cfg import config
-    from analysis_ee_Z_bb_FCCSW_cfg import config
+    
     from heppy.test.plot_ee_b import Plotter
     from heppy.framework.looper import Looper
+    from heppy.configuration import PapasRuntype
     from ROOT import TFile
-
+    PapasRuntype.from_fccsw = False
```

Comment:
why is this needed? from_fccsw should be set to False by default.

Reply:
I have removed it.
Being over cautious...